book/contributions/tec_iupac_units_excel/techniques/tec_iupac_units_excel.md

```diff
@@ -0,0 +1,83 @@
+
+There is a lot of information in the exhaustive tables in the Green Book, and it covers, not surprisingly, a wide 
+range of quantities and units across the many subdisciplines of chemistry. As a guide for humans to read it fine but
+enabling practical usage in the digital laboratory it does not, where units are almost an afterthought (IUPAC is working
+on a digital version of the Green Book as you read this.)  So, enabling the use of the recommended units in chemistry
+has been reimagined in this [Excel spreadsheet](../files/tec_iupac_units_excel.xlsx).
+
+## Using the IUPAC Units Spreadsheet
+The spreadsheet has been developed to provide the quantities and units expressed in the Green Book along with additional
+columns that characterise each unit and representations in some different encodings of units of measurement (see
+the figure below).
+
+![fig1](../images/tec_iupac_units_excel_fig1.jpg)
+
+Making units of measurement available in this spreadsheet enables it to be used to support many researchers.  In fact,
+as the representations are standardized, annotating the data with units can be considered as using a "controlled 
+vocabulary" for units.  This, plus a statement in the research data Excel file that the units were annotated using the
+IUPAC Units Spreadsheet (with a link and version #) enables others to be able to interpret the data correctly.
+
+To use the spreadsheet researchers either include the units and prefixes sheets in their data spreadsheet, OR reference
+to the spreadsheet when they add the units/prefixes to their data spreadsheet (in this case the location of the IUPAC
+Units spreadsheet should be constant).  Then, using the validation function in Excel, cells in the data spreadsheet that
+should contain units (or prefixes) are validated (using the Validation menu item under the Data menu) against the list 
+of units (see the spreadsheet for the detailed function). Then, when a user goes to each unit/prefix cell they get a 
+dropdown menu of choices (see figure below).
+
+![fig2](../images/tec_iupac_units_excel_fig2.jpg)
+
+The "Example" sheet above, included in the IUPAC Units spreadsheet, demonstrates how to use the unit and prefix lists
+to validate the content of cells. It also uses the VLOOKUP function to then lookup values for the molar masses and the
+factors associated with the prefixes.  This enables the calculation performed in the sheet such that changing the solute 
+and the numeric values for the mass and volume, the final concentration can be calculated.
+
+```{note} Challenge!
+The way that the IUPAC Units spreadsheet is used in the "Example" sheet assumes that the user knows what units are valid
+for a particular quantity.  However, this many not always be true especially if the data is being reused by another
+researcher.  How could the "Example" sheet be upgraded to make the choices available for units based on the quantity?
```

Comment:
Is this being posed as homework or we inviting feedback?

Reply:
Yes, this is challenge to the reader to make the spreadsheet even more useful.  I will clarify in an update.